content/blog/2019/08/2019-08-21-introduce-react-boilerplate-for-jenkins-plugin.adoc

```diff
@@ -0,0 +1,195 @@
+---
+layout: post
+title: "Introduce React Boilerplate for Jenkins Plugin"
+tags:
+- react
+- plugins
+- boilerplate
+- gsoc
+- gsoc2019
+author: jackshen
+---
+
+This boilerplate is part of the project link:https://summerofcode.withgoogle.com/projects/#6112735123734528[Working Hours UI Improvement] during
+link:https://summerofcode.withgoogle.com/[Google Summer of Code 2019], which improve the UI of Working Hours Plugin using this pattern to develop Jenkins plugin with React. The main repository could be found at link:https://github.com/jenkinsci/working-hours-plugin[Working Hours Plugin].
+
```

Comment:
Would be nice to have a project link in the beginning: https://github.com/ShenJack/ReactBoilerplateforJenkinsPlugin

Reply:
Yeah, I'd like it if jenkins host this, maybe I can do the transfer right now, would it be complicated?

Reply:
Please transfer to https://github.com/orgs/jenkinsci-transfer , and then I can transfer it to jenkinsci.
HOSTING request would be appreciated anyway